lib/shared/picker_item.dart

```diff
@@ -50,8 +50,6 @@ class PickerItem<T> extends StatelessWidget {
                     style: (textTheme?.bodyMedium ?? theme.textTheme.bodyMedium)?.copyWith(
                       color: (textTheme?.bodyMedium ?? theme.textTheme.bodyMedium)?.color?.withOpacity(0.5),
                     ),
-                    softWrap: false,
-                    overflow: TextOverflow.fade,

```

Comment:
Why this change? I think it was done to support long URLs in the share sheet (but I could be wrong).

| ![image](https://github.com/thunder-app/thunder/assets/7417301/55e295b1-e02e-43e2-8197-459ebed273aa) |
| - |

Reply:
Oh I see - I removed that because the descriptions of the different push notification types was being cut off 😅

![image](https://github.com/thunder-app/thunder/assets/30667958/c3971b1e-d7ba-48be-a740-f4d3f58c7a1c)

Reply:
Makes sense! Maybe we need a `wrap` option.

Reply:
Done in 65dee87952eae1a9cb8c8a707ebb899b735d5a4a!